.az/mshost.yaml

```diff
@@ -317,7 +317,16 @@ jobs:
 
       - script: |
           call activate pyenv
-          set vcvars=%VCINSTALLDIR%Auxiliary\Build\vcvars64.bat
+          @echo find where Visual Studio was installed and use the bundled cmake
```

Comment:
You can try this code : https://github.com/microsoft/onnxruntime-inference-examples/blob/main/.pipelines/OneBranch.Official.yml#L114